src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -417,6 +417,34 @@ def test_handle_job_logs_msg(self):
         msg = self.jc._comm.last_message
         self.assertEqual(msg["data"]["msg_type"], "job_logs")
 
+    def _check_rq_equal(self, rq0, rq1):
+        self.assertEqual(rq0.msg_id, rq1.msg_id)
+        self.assertEqual(rq0.rq_data, rq1.rq_data)
+        self.assertEqual(rq0.request, rq1.request)
+        self.assertEqual(rq0.job_id, rq1.job_id)
+
+    def test_split_by_job_id(self):
```

Comment:
Could you add in an "integration"-type test, where you run one of the methods with a job_id_list?